llvm/docs/RISCVUsage.rst

```diff
@@ -100,6 +100,8 @@ on support follow.
      ``V``            Supported
      ``Za128rs``      Supported (`See note <#riscv-profiles-extensions-note>`__)
      ``Za64rs``       Supported (`See note <#riscv-profiles-extensions-note>`__)
+     ``Zaamo``        Supported
```

Comment:
This is incorrect if this the MC layer patch only Assembly Support is included. The top of this section also says "Ratified Extensions by Status" so it shouldn't even be in this list.